include/rocksdb/perf_level.h

```diff
@@ -14,19 +14,39 @@
 namespace ROCKSDB_NAMESPACE {
 
 // How much perf stats to collect. Affects perf_context and iostats_context.
+// These levels are incremental, which means a new set of metrics will get
+// collected when PerfLevel is upgraded from level n to level n + 1.
+// Each level's documentation specifies the incremental set of metrics it
+// enables. As an example, kEnableWait will also enable collecting all the
+// metrics that kEnableCount enables, and its documentation only specifies which
+// extra metrics it also enables.
+//
+// These metrics are identified with some naming conventions, but not all
+// metrics follow exactly this convention. The metrics' own documentation should
+// be source of truth if they diverge.
 enum PerfLevel : unsigned char {
-  kUninitialized = 0,             // unknown setting
-  kDisable = 1,                   // disable perf stats
-  kEnableCount = 2,               // enable only count stats
-  kEnableWait = 3,                // measure time spent by user threads
-                                  // blocked in RocksDB, and not external
-                                  // resources such as mutexes and IO
-  kEnableTimeExceptForMutex = 4,  // Other than count stats, also enable time
-                                  // stats except for mutexes
-  // Other than time, also measure CPU time counters. Still don't measure
-  // time (neither wall time nor CPU time) for mutexes.
+  // Unknown setting
+  kUninitialized = 0,
+  // Disable perf stats
+  kDisable = 1,
+  // Starts enabling count metrics. These metrics usually don't have time
+  // related keywords, and are likely to have keywords like "count" or "byte".
+  kEnableCount = 2,
+  // Starts enabling metrics that measure time spent by user threads blocked in
+  // RocksDB waiting for RocksDB to take actions, as opposed to waiting for
+  // external resources such as mutexes and IO.
+  // These metrics usually have this pattern: "_[wait|delay]_*_[time|nanos]".
+  kEnableWait = 3,
+  // Starts enabling metrics that measure the end to end time of an operation.
+  // These metrics' names have keywords "time" or "nanos". Check other time
```

Comment:
nit: "time" or time unit e.g, nanos, micros.

Reply:
Thanks for the suggestion! I specifically used "nanos" because that's the unit used for for all the time measuring perf context: https://github.com/facebook/rocksdb/blob/1d919ac414141683bac75775b76ea06dab71bfed/monitoring/perf_step_timer.h#L60-L66